rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -20,6 +20,9 @@
 sys.path.insert(0, jedi_path)
 import jedi  # noqa: E402
 
+# Todo: workaround for Python 3.7
+jedi.api.environment._SUPPORTED_PYTHONS.insert(0, '3.7')
```

Comment:
This should handle the case for when it gets added (https://github.com/davidhalter/jedi/pull/1217), i.e. only add it if not there yet (although we control this by updating the submodule ourselves currently, but might be forgotten also anyway).